js/tuxexchange.js

```diff
@@ -0,0 +1,533 @@
+        const currencyIds = Object.keys (currenciesData);
+        // The API does not expose BTC as a coin (I suspect because it is the base in each market)
+        let result = { 'BTC': {
+            'id': 'BTC',
+            'code': 'BTC',
+            'name': 'bitcoin',
+            'fiat': false,
+        }};
+        for (let i = 0; i < currencyIds.length; i++) {
+            const id = currencyIds[i];
+            const coinData = currenciesData[id];
+            const code = this.commonCurrencyCode (id);
+            result[code] = {
+                'id': id,
+                'code': code,
+                'name': coinData['name'],
+                'fiat': false,
+                'funding': {
+                    'withdraw': {
+                        'fee': this.safeFloat (coinData, 'withdrawfee'),
+                    },
+                },
+            };
+        }
+        return result;
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const tickersData = await this.publicGet (this.extend ({ 'method': 'getticker' }, params));
+        const tickerIds = Object.keys (tickersData);
+        const result = {};
+        for (let i = 0; i < tickerIds.length; i++) {
+            const id = tickerIds[i];
+            const tickerResult = this.parseTicker (id, tickersData[id]);
+            const symbol = tickerResult['symbol'];
+            result[symbol] = tickerResult;
+        }
+        return result;
+    }
+
+    parseTicker (marketId, tickerData) {
+        const timestamp = this.milliseconds ();
+        const market = this.findMarket (marketId);
+        return {
+            'symbol': market['symbol'],
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'close': this.safeFloat (tickerData, 'last'),
+            'last': this.safeFloat (tickerData, 'last'),
+            'high': this.safeFloat (tickerData, 'high24hr'),
+            'low': this.safeFloat (tickerData, 'low24hr'),
+            'percentage': this.safeFloat (tickerData, 'percentChange'),
+            'baseVolume': this.safeFloat (tickerData, 'baseVolume'),
+            'quoteVolume': this.safeFloat (tickerData, 'quoteVolume'),
+            'bid': this.safeFloat (tickerData, 'highestBid'),
+            'ask': this.safeFloat (tickerData, 'lowestAsk'),
+            'bidVolume': undefined,
+            'askVolume': undefined,
+            'vwap': undefined,
+            'open': undefined,
+            'previousClose': undefined,
+            'change': undefined,
+            'average': undefined,
+            'info': tickerData,
+        };
+    }
+
+    async fetchOrderBook (symbol, limit = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchOrderBook () requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        const codes = this.getIdsFromSymbol (symbol);
```

Comment:
We should remove this method from everywhere, and do this: 

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#sending-market-ids
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#parsing-symbols